Fix/webpages reorganized

PrajwalaMugajjiShambulingappa opened the pull request:
This PR address website address minor UI adjustments to improve visual alignment.

Crustaly reviewed (commented):
looks good! @master-of-none could you check this out as well?

PrajwalaMugajjiShambulingappa commented:
Hey @jschhie!
Thanks for flagging these! I’ve gone ahead and fixed the following in:
- Updated your last name to “Chhie”
- Switched to jennifer.png
- Updated your role to “Developer”
- Noted Carter Sargent as “Accessibility Specialist” 

Let me know if anything else needs tweaking, Thanks in advance!

master-of-none reviewed (approved):
Merging this PR to preprod
